frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/create-local-volume-set.tsx

```diff
@@ -63,6 +76,9 @@ export const CreateLocalVolumeSet: React.FC<CreateLocalVolumeSetProps> = ({
             dispatch={dispatch}
             diskModeOptions={diskModeDropdownItems}
             allNodesHelpTxt={allNodesSelectorTxt}
+            taintsFilter={hasOCSTaint}
+            setTaintedNodes={setOCSTaint}
+            taintedNodes={state.taintedNodesForADV}
```

Comment:
This should not be required if we just set it based on the component being called from OCS.

Reply:
Can you elaborate ?
Not following. Thanks!

Reply:
Probably we can have a simple solution of having tolerations set if the LVS & ADV created from the OCS component and not look got the tainted nodes are there or not .

Reply:
This means we are downgrading, since the current logic is generic and not very OCS specific. If anytime later, we support taints for LSO, then we just need to remove the props (alongwith any UX changes)